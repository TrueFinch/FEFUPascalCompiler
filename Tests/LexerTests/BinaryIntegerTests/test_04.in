%100             %0101   
%1  %001             %110            
  %010101010101   %0 %1   
     %01        %01     01    
 %111      %1    
  %001       
 %1    %001  %0101010100101010101       %1
     
                    %1
                                     
 %1011      %1 
  %10                         %010101  %1