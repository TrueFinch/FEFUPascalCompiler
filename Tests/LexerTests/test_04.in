%101010 %10000 %001