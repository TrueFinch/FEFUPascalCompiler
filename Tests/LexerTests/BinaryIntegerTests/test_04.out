1,1 TYPE_INTEGER 4 %100
1,18 TYPE_INTEGER 5 %0101
2,1 TYPE_INTEGER 1 %1
2,5 TYPE_INTEGER 1 %001
2,22 TYPE_INTEGER 6 %110
3,3 TYPE_INTEGER 1365 %010101010101
3,19 TYPE_INTEGER 0 %0
3,22 TYPE_INTEGER 1 %1
4,6 TYPE_INTEGER 1 %01
4,17 TYPE_INTEGER 1 %01
4,25 TYPE_INTEGER 1 01
5,2 TYPE_INTEGER 7 %111
5,12 TYPE_INTEGER 1 %1
6,3 TYPE_INTEGER 1 %001
7,2 TYPE_INTEGER 1 %1
7,8 TYPE_INTEGER 1 %001
7,14 TYPE_INTEGER 174421 %0101010100101010101
7,41 TYPE_INTEGER 1 %1
9,21 TYPE_INTEGER 1 %1
11,2 TYPE_INTEGER 11 %1011
11,13 TYPE_INTEGER 1 %1
12,3 TYPE_INTEGER 2 %10
12,31 TYPE_INTEGER 21 %010101
12,40 TYPE_INTEGER 1 %1
